Add diki run script in /hack (#120)
gardener/diki: README.md

#### Installation

If you install via GitHub releases, you need to put the diki binary on your path.

> You can also install diki using this code snippet:

Reviewer: We can remove this line

> You can also install diki using this code snippet:

Reviewer: ```suggestion
A sample install snippet for macOS can look like this:
```
